docs/language_specification/tokens/operators_and_punctuators.md

```diff
@@ -0,0 +1,11 @@
+The lexical representation of cQASM programs includes a number of tokens which serve the function of operators and punctuators:
```

Comment:
```suggestion
The lexical representation of cQASM includes tokens that serve as operators or punctuators.
```
What is meant by 'The lexical representation of cQASM programs'?

Reply:
I've kept the last part of the suggestion.